<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CUserVariablView</class>
 <widget class="QMainWindow" name="CUserVariablView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CUserVariablView</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>filter column</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="comboBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <item>
       <property name="text">
        <string>ID</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>TagName</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Description</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>OccNo</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DataType</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>InitValue</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>InitialQua</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Address</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PinLabel</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>SourceTagName</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Scale</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>filter string</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item row="1" column="0" colspan="5">
     <widget class="CUserVariableTable" name="tableView"/>
    </item>
    <item row="0" column="4">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>145</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>643</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CUserVariableTable</class>
   <extends>QTableView</extends>
   <header>uservariabletable.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
